pymeasure/instruments/racal/racal1992.py

```diff
@@ -80,26 +84,27 @@ class Racal1992(Instrument):
         'frequency_c'     : 'FC',    # noqa
     }
 
-    # All values returned follow the same format: 2 letters to indicate
-    # the type of the value returned, followed by a floating point number
-    # (which could be an integer, of course.)
-    # This here, for example, is math constant Z: MZ+001.00000000E+00
     def decode(v, allowed_types=None):
+        """All values returned follow the same format: 2 letters to indicate
+        the type of the value returned, followed by a floating point number
+        (which could be an integer, of course.)
+        This here, for example, is math constant Z: MZ+001.00000000E+00
+        """
```

Comment:
To follow python standards, a docstring should start with an imperative first clause:
```suggestion
        """Decode received message.

        All values returned follow the same format: 2 letters to indicate
        the type of the value returned, followed by a floating point number
        (which could be an integer, of course.)
        This here, for example, is math constant Z: MZ+001.00000000E+00
        """
```